### PR TITLE
Fix: Address markdown syntax issues flagged by linting checks

### PR DESCRIPTION
**Pull request by ModeSevenIndustrialSolutions**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,61 +19,64 @@ The action extracts the commits from a GitHub pull-request and submits them to a
...
+Or, submit each commit as a separate single commit preserving the git history (SUBMIT_SINGLE_COMMITS).
 
-3. Check for a Change-Id line in the pull request commit message. If it is not present, add the Change-Id to the commit. If the Change-Id is found in any of the commits, it will be reused along with the patch.
-4. Add the Change-Id (and optionally squash changes into a single commit if required).
-5. Add a pull-request and workflow run reference link as a comment on the Gerrit change that was created for committers or reviewers to back reference to the source of change request.
-6. Add a comment to the pull request with the URL to the change. Any updates will require the pull request to be reopened and updates to the pull request must be done with a force push, which triggers the workflows to ensure the change is resubmitted.
-7. Close the pull request once the Gerrit patch is submitted successfully.
+1. Check for a Change-Id line in the pull request commit message. If it is not present, add the Change-Id to the commit. If the Change-Id is found in any of the commits, it will be reused along with the patch.
```
Comment (askb) [inline README.md:31]: The seq is all changed to 1.  is this expected?

File: README.md [hunk 2]
```diff
@@ -14,61 +19,64 @@ The action extracts the commits from a GitHub pull-request and submits them to a
...
+1. Add the Change-Id (and optionally squash changes into a single commit if required).
```
Comment (askb) [inline README.md:32]: The seq is all changed to 1.  is this expected?

File: README.md [hunk 3]
```diff
@@ -14,61 +19,64 @@ The action extracts the commits from a GitHub pull-request and submits them to a
...
+1. Add a pull-request and workflow run reference link as a comment on the Gerrit change that was created for committers or reviewers to back reference to the source of change request.
+1. Add a comment to the pull request with the URL to the change. Any updates will require the pull request to be reopened and updates to the pull request must be done with a force push, which triggers the workflows to ensure the change is resubmitted.
+1. Close the pull request once the Gerrit patch is submitted successfully.
```
Comment (askb) [inline README.md:35]: The seq is all changed to 1.  is this expected?

**Comment by askb** [comment]
the pre-commit errors need to be addressed.

File: README.md (hunk 1 above)
Comment (ModeSevenIndustrialSolutions) [reply README.md:31]: Yes, incrementing the numbering happens automatically. It makes it much easier to re-order content.

File: README.md (hunk 2 above)
Comment (ModeSevenIndustrialSolutions) [reply README.md:32]: See here: https://experienceleague.adobe.com/en/docs/contributor/contributor-guide/writing-essentials/markdown
```
Numbered lists and bullet lists
To create numbered lists, begin a line with 1. or 1), but don’t use both formats within the same list. You don’t need to specify the numbers. GitHub does that for you.

1. This is step 1.
1. This is the next step.
1. This is yet another step, the third.
```

File: README.md (hunk 3 above)
Comment (ModeSevenIndustrialSolutions) [reply README.md:35]: See above, yes.

**Review by askb — APPROVED** [review]

**Pull request merged**